IdentityCore/src/configuration/MSIDNetworkConfiguration.m

```diff
@@ -22,29 +22,37 @@
 // THE SOFTWARE.
 
 #import "MSIDNetworkConfiguration.h"
+#import "MSIDURLSessionManager.h"
 
-static NSTimeInterval s_timeout = 30;
-static NSInteger s_retryCount = 2;
+static NSInteger s_retryCount = 1;
+static NSTimeInterval s_retryInterval = 0.5;
 
 @implementation MSIDNetworkConfiguration
 
-+ (void)setTimeout:(NSTimeInterval)timeout
++ (void)setTimeoutIntervalForRequest:(NSTimeInterval)timeoutIntervalForRequest
 {
-    s_timeout = timeout;
+    MSIDURLSessionManager.defaultManager.configuration.timeoutIntervalForRequest = timeoutIntervalForRequest;
 }
-+ (NSTimeInterval)timeout
+
++ (void)setTimeoutIntervalForResource:(NSTimeInterval)timeoutIntervalForResource
 {
-    return s_timeout;
+    MSIDURLSessionManager.defaultManager.configuration.timeoutIntervalForResource = timeoutIntervalForResource;
 }
 
-+(void)setRetryCount:(NSInteger)retryCount
++ (NSTimeInterval)timeoutIntervalForResource
 {
-    s_retryCount = retryCount;
+    return MSIDURLSessionManager.defaultManager.configuration.timeoutIntervalForResource;
 }
 
-+ (NSInteger)retryCount
++ (NSTimeInterval)timeoutIntervalForRequest
 {
-    return s_retryCount;
+    return MSIDURLSessionManager.defaultManager.configuration.timeoutIntervalForRequest;
 }
 
++ (void)setRetryCount:(NSInteger)retryCount { s_retryCount = retryCount; }
```

Comment:
should those one be in MSIDURLSessionManager as well, so we can get rid of this class?

Reply:
sure, it's internal, so let's spread the settings to HTTPRequest and url session manager